### PR TITLE
Added sign up and login pages

### PR DESCRIPTION
**Pull request by nimish-keel**
Social Login functionality is under implementation.

And storing auth token after successfully logging or sign up is under implementation.

### CODE REVIEW

File: dev/js/actions/commons/login.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    .then(function (response) {
+      if (response && response.status == 1) {
+        dispatch({
+          type: SEND_USER_LOGIN_CREDENTIALS,
+          payload: {
+            loading: false,
+          },
+        });
+        if (cb) cb(null, response);
+      }
```
Comment (Pankajkeel) [inline dev/js/actions/commons/login.js:28]: Plz add else condition also if the status is not 1 or response not exist we will show some error, also ask shubh to give some error screen or toaster

File: dev/js/actions/commons/login.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    })
+    .then(function () {
```
Comment (Pankajkeel) [inline dev/js/actions/commons/login.js:30]: there is no need for this ".then" you can do the same work in above then statement, you can club both dispatch in above ".then"

File: dev/js/actions/commons/login.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import { API_POST } from '../../api/api.js';
+
+export const userLogin = (data, dispatch, cb) => {
+  dispatch({
+    type: SEND_USER_LOGIN_CREDENTIALS,
+    payload: {
+      loading: true,
+    },
+  });
+  API_POST('https://getkeel.com/api/v1/user/login', {
```
Comment (Pankajkeel) [inline dev/js/actions/commons/login.js:15]: Please use CONSTANT for host name "https://getkeel.com/" in all api call

File: dev/js/actions/commons/signup.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import { API_POST } from '../../api/api.js';
+
+export const userSignUp = (data, dispatch, cb) => {
+  dispatch({
+    type: SEND_USER_SIGNUP_CREDENTIALS,
+    payload: {
+      loading: true,
+    },
+  });
+  API_POST('https://getkeel.com/api/v1/user/signup', {
```
Comment (Pankajkeel) [inline dev/js/actions/commons/signup.js:15]: use CONSTANT

File: dev/js/actions/commons/signup.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+          type: SEND_USER_SIGNUP_CREDENTIALS,
+          payload: {
+            loading: false,
+          },
+        });
+        if (cb) cb(null, response);
+      }
+    })
+    .then(function () {
+      let message = 'User Signed Up Successfully';
```
Comment (Pankajkeel) [inline dev/js/actions/commons/signup.js:31]: same as above club dispatch in single ".then"

File: dev/js/api/api.js [hunk 6]
```diff
@@ -1,133 +1,143 @@
...
-    API_GET,
-    API_POST,
-    API_PUT,
-    API_DELETE
-}
+  API_GET,
+  API_POST,
+  API_PUT,
+  API_DELETE,
+};
```
Comment (Pankajkeel) [inline dev/js/api/api.js:143]: Bro Plz check your PR , why all my files that I already merged to master is showing deleted in your PR, check below deleted files, it's all required

**Review by Pankajkeel — CHANGES_REQUESTED** [review]
Create new branch & pull user changes

**Comment by nimish-keel** [comment]
obsolete

**Pull request closed without merging**